osu.Game/Overlays/BeatmapSet/Header.cs

```diff
@@ -146,6 +147,13 @@ public Header()
                                                         Origin = Anchor.BottomLeft,
                                                         Margin = new MarginPadding { Left = 3, Bottom = 4 }, // To better lineup with the font
                                                     },
+                                                    explicitPill = new ExplicitBeatmapPill
+                                                    {
+                                                        Alpha = 0f,
+                                                        Anchor = Anchor.CentreLeft,
+                                                        Origin = Anchor.CentreLeft,
+                                                        Margin = new MarginPadding { Left = 10f, Top = 4 },
+                                                    }
```

Comment:
This usage does not look great with the "open in browser" icon:

![2021-01-13-211219_246x49_scrot](https://user-images.githubusercontent.com/20418176/104505392-164d4800-55e4-11eb-87d5-2baa617dc118.png)

My particular concerns are: the wonky horizontal spacing (icon almost glued to text, while explicit label is further away), and the wonky vertical alignment (icon is aligned to text baseline, while explicit label is vertically centred). Would probably tinker with that more as it looks bad IMO.

Reply:
The external link should be glued to the right bottom of the title, it can be considered as part of the title, while the explicit pill is meant to be separate marking the beatmap as explicit, I think spacing that out would do it better.

Reply:
![image](https://user-images.githubusercontent.com/22781491/104537263-b3cb6a80-562a-11eb-9e73-92f46fdef0e7.png)

How does this look?

Reply:
Maybe sliiiightly better but still not completely sold on it. Might be just me though?

Reply:
Upon re-checking web, the explicit badge is also closer to baseline than vertically centered. I think something like this looks much better:

![2021-01-14-223417_245x71_scrot](https://user-images.githubusercontent.com/20418176/104652012-c0969f80-56b8-11eb-80e2-91181394b360.png)

Code for this is:

```csharp
explicitPill = new ExplicitBeatmapPill
{
    Alpha = 0f,
    Anchor = Anchor.BottomLeft,
    Origin = Anchor.BottomLeft,
    Margin = new MarginPadding { Left = 8, Bottom = 4 },
}
```

Reply:
Seems fine for me.